nextflow_schema.json

```
@@ -374,6 +374,40 @@
                     "fa_icon": "fas fa-save",
                     "description": "Specify to save intermediate temporary files to --outdir",
                     "help_text": "RGI produces a lot of additional temporary files which in most cases won't be useful.\n\nBy specifying this parameter, the files including `temp` in the name will be also saved in the output directory for closer inspection by the user, if necessary."
+                },
+                "arg_rgi_alignmenttool": {
+                    "type": "string",
+                    "default": "BLAST",
+                    "description": "Specifies the alignment tool to be used.",
+                    "help_text": "Specifies the alignment tool to be used. By default the tool used for alignment is `DIAMOND`. This flag activates `BLAST` instead.\n\n> Modifies tool parameter(s):\n> - RGI: `--alignment_tool`",
+                    "enum": ["BLAST", "DIAMOND"],
+                    "fa_icon": "fas fa-align-justify"
+                },
+                "arg_rgi_includeloose": {
+                    "type": "boolean",
+                    "description": "Includes all of loose, strict and perfect hits (i.e. >=95% identity) found by rgi.",
+                    "help_text": "When activated it includes all of loose, strict and perfect hits (i.e. also those <=95% identity) found by rgi, and list those hits still as 'strict'.\n\n> Modifies tool parameter(s):\n> - RGI: `--include_loose`",
```

The behaviour that RGI includes the loose hits with an identity of 95% or better as `strict` is from versions previous to 6.0.0. according to the documentation on git (https://github.com/arpcard/rgi#help-menus-for-subcommands). The module still uses v.5.2.1.  Should we update the module at some point?

```suggestion
                    "help_text": "When activated it includes loose hits (a.k.a. Discovery) in addition to strict and perfect hits. All Loose matches of 95% identity or better are automatically listed as Strict, regardless of alignment length (RGI v. <6.0.0). This behaviour can be overrun by using the --exclude_nudge flag. The Loose algorithm works outside of the detection model cut-offs to provide detection of new, emergent threats and more distant homologs of AMR genes, but will also catalog homologous sequences and spurious partial matches that may not have a role in AMR  \n\n> Modifies tool parameter(s):\n> - RGI: `--include_loose`",
```

---

Indeed, it seems like they made a major update of RGI, thats why I made an issue for that. Perhaps after the first release?